### PR TITLE
rancher-turtles: release 0.3.0

### PR DESCRIPTION
**Pull request by hardys**
- Use image tag instead of digest, because  Hauler doesn't handle the multi-arch digest currently
- Update RKE2 provider to 0.7.0 release
- Disable pre-install and pre-delete hooks as a workaround for issues described in https://github.com/rancher/turtles/issues/627
- Add missing Namespace definitions to rancher-turtles-airgap-resources

### CODE REVIEW

File: charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      type:         "CoreProvider"
+
+    # Infrastructure providers
+    - name:         "metal3"
+      url:          "https://github.com/metal3-io/cluster-api-provider-metal3/releases/v1.7.1/infrastructure-components.yaml"
+      type:         "InfrastructureProvider"
+
+    # Bootstrap providers
+    - name:         "rke2"
+      url:          "https://github.com/rancher/cluster-api-provider-rke2/releases/v0.6.1/bootstrap-components.yaml"
```
Comment (fdegir) [inline charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml:22]: Shouldn't we bump `cluster-api-provider-rke2` to `v0.7.0`?

File: charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      type:         "BootstrapProvider"
+
+    # ControlPlane providers
+    - name:         "rke2"
+      url:          "https://github.com/rancher/cluster-api-provider-rke2/releases/v0.6.1/control-plane-components.yaml"
```
Comment (fdegir) [inline charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml:27]: Shouldn't we bump `cluster-api-provider-rke2` to `v0.7.0`?

File: charts/rancher-turtles/0.3.0+up0.11.0/values.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      imageUrl: "registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-controller:1.7.5"
+      fetchConfig:
+        url: ""
+        selector: ""
+    rke2:
+      enabled: true
+      version: ""
+      bootstrap:
+        namespace: rke2-bootstrap-system
+        imageUrl: "registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-rke2-bootstrap:0.6.1"
```
Comment (fdegir) [inline charts/rancher-turtles/0.3.0+up0.11.0/values.yaml:65]: Shouldn't we bump `cluster-api-provider-rke2-bootstrap` to `v0.7.0`?

File: charts/rancher-turtles/0.3.0+up0.11.0/values.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        fetchConfig:
+          url: ""
+          selector: ""
+      controlPlane:
+        namespace: rke2-control-plane-system
+        imageUrl: "registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-rke2-controlplane:0.6.1"
```
Comment (fdegir) [inline charts/rancher-turtles/0.3.0+up0.11.0/values.yaml:71]: Shouldn't we bump `cluster-api-provider-rke2-controlplane` to `v0.7.0`?

**Review by fdegir — COMMENTED** [review]
Comments inline regarding v0.7.0 version bump for RKE2 CAPI provider.

File: charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml (hunk 1 above)
Comment (hardys) [reply charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml]: Updated, thanks!

File: charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml (hunk 2 above)
Comment (hardys) [reply charts/rancher-turtles/0.3.0+up0.11.0/templates/clusterctl-config.yaml]: Updated, thanks!

File: charts/rancher-turtles/0.3.0+up0.11.0/values.yaml (hunk 3 above)
Comment (hardys) [reply charts/rancher-turtles/0.3.0+up0.11.0/values.yaml]: Updated, thanks!

**Review by Kristian-ZH — APPROVED** [review]
Worked in Airgap env

**Comment by hardys** [comment]
Feedback via slack from @Kristian-ZH and @fdegir indicates this is now ready to merge

**Pull request merged**